pkgs/development/libraries/gtk/patches/2.0-sysconfdir_install.patch

```diff
@@ -0,0 +1,470 @@
+diff --git a/build/win32/vs10/gtk+.sln b/build/win32/vs10/gtk+.sln
+index 57cba1d8b2..8224d0b9a7 100644
+--- a/build/win32/vs10/gtk+.sln
```

Comment:
Why is this file even being patched?

Reply:
Oh, because when closing the git repo the line endings changed.